kbase-extension/static/kbase/js/widgets/appWidgets2/input/checkboxInput.js

```diff
@@ -8,33 +8,33 @@ define([
 
     'bootstrap',
     'css!font-awesome'
-], function(
+], (
```

Comment:
Ran eslint + prettier on this file. I edited the content and have commented below on what I did.